### DISCUSSION TITLE
Passing "props" to custom node views

### DISCUSSION THREAD

**Discussion by jondavidpass** [ueberdosis/tiptap#2507] [General Questions] [answered]
Is there a recommended way to pass data to a node view? 

I was either thinking of doing something like this:
```
this.editor = new Editor({ ... });
this.editor.myProperty = this.localProperty;
```

I am using Vue 3 so maybe provide / inject would be a better plan?

Any insight would be appreciated 🙏

**Comment by philippkuehn** [comment, marked as answer]
Using `provide/inject` is the recommended way of passing props when using Vue. There is no API to pass custom data right now.

**Reply by jondavidpass** [reply]
Makes sense. Thanks!
